src/example_posterior_parameter_sets.txt

```diff
@@ -0,0 +1,101 @@
+iterID,nsse_cases,nsse_deaths,p_inf,p_hcw,c_hcw,d,q,p_s,rrd,intro,weight
```

Comment:
This file should be moved to the `data` directory. The `src` directory is for code.

The same comment applies to the other CSV file.